From ccb00292eb2dbb58a55850639356d07630cd3c46 Mon Sep 17 00:00:00 2001
Message-Id: <ccb00292eb2dbb58a55850639356d07630cd3c46.1648731916.git.geliang.tang@suse.com>
From: Christoph Hellwig <hch@lst.de>
Date: Fri, 19 Nov 2021 17:32:12 +0100
Subject: [PATCH] bpf: Remove a redundant comment on bpf_prog_free
Git-commit: ccb00292eb2dbb58a55850639356d07630cd3c46
Patch-mainline: v5.17-rc1
References: bsn#127
Modified-by-SEL: No


The comment telling that the prog_free helper is freeing the program is
not exactly useful, so just remove it.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Acked-by: Song Liu <songliubraving@fb.com>
Link: https://lore.kernel.org/bpf/20211119163215.971383-3-hch@lst.de
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 kernel/bpf/core.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/bpf/core.c b/kernel/bpf/core.c
index b52dc845ecea..189d85d64bf1 100644
--- a/kernel/bpf/core.c
+++ b/kernel/bpf/core.c
@@ -2301,7 +2301,6 @@ static void bpf_prog_free_deferred(struct work_struct *work)
 	}
 }
 
-/* Free internal BPF program */
 void bpf_prog_free(struct bpf_prog *fp)
 {
 	struct bpf_prog_aux *aux = fp->aux;
-- 
2.34.1

